doc/content/concepts/devicetemplates/creating.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Creating Templates"
+description: ""
+weight: 2
+---
+
+You can create a template from scratch, set fields in an existing template or create a template an existing device.
+
+## Create from scratch
+
+Use the `device template create` command to create a template with some end device fields set:
+
+```bash
+$ ttn-lw-cli device template create \
+  --lorawan-version 1.0.3 \
+  --lorawan-phy-version 1.0.3-a \
+  --mapping-key test
+```
+
+<details><summary>Show output</summary>
+```json
+{
+  "end_device": {
+    "ids": {
+      "application_ids": {
+
+      }
+    },
+    "created_at": "0001-01-01T00:00:00Z",
+    "updated_at": "0001-01-01T00:00:00Z",
+    "attributes": {
+    },
+    "lorawan_version": "1.0.3",
+    "lorawan_phy_version": "1.0.3-a"
+  },
+  "field_mask": {
+    "paths": [
+      "lorawan_phy_version",
+      "lorawan_version"
+    ]
+  },
+  "mapping_key": "test"
+}
+```
+</details>
+
+See `$ ttn-lw-cli device template create --help` for all the fields that can be set.
+
+>Note: The **mapping key** is optional. Mapping keys can be used to map templates with each other. See [Mapping Templates](../mapping).
+
+## Set fields
+
+The `device template create` command may take an existing template as input. This allows you to set fields in an existing template.
+
+For example, create the base template and save to `base.json`:
+
+```bash
+$ ttn-lw-cli device template create \
+  --lorawan-version 1.0.3 \
+  --lorawan-phy-version 1.0.3-a \
+  --mapping-key test > base.json
```

Comment:
I think having the mapping key in this particular example is a bit confusing. Maybe leave that for `mapping.md`?